cmd/promtool/main.go

```diff
@@ -20,8 +20,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
I don't like us introducing random libraries where we've already established a defacto standard within the codebase. That way does not lie sanity.